src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/AppearancePreferencesPane.R

```diff
@@ -15,7 +17,7 @@ fivenum <- function(x) {
   i <- c(n5, n4, n3, n2, n1)
   
   # compute quartile values
-  x <- sort(x)
+  x <- base::sort(x)
```

Comment:
This seems kind of arbitrary; I think the preview code we use should reflect "real" code as much as possible, and I don't think prefixing with `base::` is common. I'm not sure if there's a better alternative. I think it's okay if we omit usages of `::` in the preview here.

Reply:
I added a default of `stats::rnorm(100)`, but if you are not convinced, that's okay, we can omit this.

I also added `decreasing = FALSE` to highlight how logical are shown